examples/02-consume/src/main.rs

```diff
@@ -47,7 +47,7 @@ async fn consume() -> Result<(), fluvio::FluvioError> {
     let mut stream = consumer.stream(fluvio::Offset::beginning()).await?;
 
     while let Some(Ok(record)) = stream.next().await {
-        let bytes = record.as_ref();
+        let bytes = record.value();
         let string = String::from_utf8_lossy(&bytes);
```

Comment:
Why this need ref since value returns ref

Reply:
I think the general pattern I see in rust is the following naming convention:
- `value(&self)` the simple name takes self by immutable reference
- `value_mut(&mut self)` the `_mut` name takes self by mutable reference
- `into_value(self)` the `into_` name consumes self by ownership

so I think `value` is the right choice here

Reply:
Actually, it was not about name.   
```
String::from_utf8_lossy(&bytes);
```
Since value() is ref, shouldn't be:
```
String::from_utf8_lossy(bytes);
```

Reply:
Oh I see, you're right, I will fix that

Reply:
Fixed